Assignment-03/assignment 03-1.3 .c

```diff
@@ -0,0 +1,13 @@
+#include<stdio.h>
+//author sidhant
+//date of creation
+int main()
+{
+    char str1[10],str2[10];
+    printf("enter the first name \n");    //input the first name
+    scanf("%s",&str1);
+    printf("enter the last name \n");      //input last name
+    scanf("%s",&str2);
+    printf("hello %s %s",&str1,&str2);

```

Comment:
Missing newline in _printf_.

Reply:
ok sir i really appreciate your efforts to have a look at my report and thanks for the honest reviews